### DISCUSSION TITLE
Is there a way to just use rich editor of AppFlowy only?

### DISCUSSION THREAD

**Discussion by lanistor** [AppFlowy-IO/AppFlowy#582] [General]

**Comment by annieappflowy** [comment]
We're developing the flowy editor, which could be used as a standalone component. I'd love to know more about your requirements. Can you please share?

**Reply by omarelb** [reply]
Hi there!

It is awesome to hear that you are working on this, as this is something I would love to see!

I'm developing a recipe application in which you for example are able to save recipes and edit them. I really like how editing in Notion works, so I wanted to bring that in as well. For me the most important features would be:
- Add new blocks of different types (mainly headers or normal text and easily remove / reorder  blocks and open overlay settings menu for a block
- Extensibility: the ability to define a custom block type with an associated customizable rendering view and data model. If this is too difficult, a way to assign semantic information to a certain block could also work. In case of the recipe app for example, I would want to add an "ingredient", and the editor data model should somehow indicate that this block is an ingredient.

Really looking forward to this! I'd be happy to give more thoughts if necessary:)

Cheers

**Reply by annieappflowy** [reply]
Hi @omarelb , thanks a lot for you input, super valuable to us. 
We'll be sharing the design doc on GitHub soon, and welcome feedback and more input!
